### PR TITLE
add getTargetByName to MorphTargetManager

### PR DESCRIPTION
**Pull request by deltakosh**
https://forum.babylonjs.com/t/animatable-not-working/50775/15

### CODE REVIEW

**Review by sebavan — APPROVED** [review]

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15134/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15134/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15134/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15134/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15134/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15134/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15134/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15134/merge#BCU1XR#0

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15134/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15134/merge/testResults/webgpuplaywright/index.html

File: packages/dev/core/src/Morph/morphTargetManager.ts [hunk 1]
```diff
@@ -230,6 +230,21 @@ export class MorphTargetManager implements IDisposable {
         return this._targets[index];
     }
 
+    /**
+     * Gets the target at specified index
```
Comment (Popov72) [inline packages/dev/core/src/Morph/morphTargetManager.ts:234]: index => name

Comment (sebavan) [reply packages/dev/core/src/Morph/morphTargetManager.ts:234]: I ll do a PR in a sec

**Pull request merged**